server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -259,7 +261,10 @@ func (r *statsRecorder) Start() {
 				// for any other purpose).
 				hit_tracker.CleanupCacheStats(ctx, r.env, task.invocationJWT.id)
 				if !updated {
-					return status.CanceledErrorf("Attempt %d of invocation %s pre-empted by more recent attempt, no cache stats flushed.", task.invocationJWT.attempt, task.invocationJWT.id)
+					log.Debugf("Attempt %d of invocation %s pre-empted by more recent attempt, no cache stats flushed.", task.invocationJWT.attempt, task.invocationJWT.id)
```

Comment:
We don't have Debug logs enabled in prod. If you want to know whether it's happening in prod, you'd need to raise to at least Info.

Reply:
It looks like we use `Warningf` in other cases where invocations are pre-empted, so I changed this to Warning level. (Though we might want to reconsider downgrading these to Debug level once we are more confident that we handle retries correctly)